Actions: Error in the CI

Due to a recent update in **pylint** ( >= v2.13.0 ), some linting errors popped up in the Actions checker.

```
************* Module kcidb.mq
mq/__init__.py:552:4: W0221: Number of parameters was 1 in 'Subscriber.decode_data' and is now 2 in overridden 'EmailSubscriber.decode_data' method (arguments-differ)
************* Module kcidb.oo
oo/__init__.py:349:12: E4702: Iterated dict 'repo_branch_checkouts' is being modified inside for loop body, iterate through a copy of it instead. (modified-iterating-dict)
```

Possible options to solve this issue
- Set a specific version of pylint(<2.13.0)
- We can also disable the pylint checker
- To solve the `modified-iterating-dict`, use `repo_branch_checkout.items()` instead of `for repo in repo_branch_checkout`.

or others...

--- a/kcidb/mq/__init__.py
+++ b/kcidb/mq/__init__.py
@@ -549,6 +549,7 @@ class EmailPublisher(Publisher):
 class EmailSubscriber(Subscriber):
     """Email queue subscriber"""
 
+    # We'll be OK, pylint: disable=arguments-differ
     def decode_data(self, message_data):
         """
         Decode email from the message data.

--- a/kcidb/oo/__init__.py
+++ b/kcidb/oo/__init__.py
@@ -344,6 +344,7 @@ class Revision(Object, BuildContainer, TestContainer):
             if branch not in branch_checkouts:
                 branch_checkouts[branch] = []
             branch_checkouts[branch].append(checkout)
+        # pylint: disable=modified-iterating-dict
         # Sort branches by number of checkouts
         for repo in repo_branch_checkouts:
             repo_branch_checkouts[repo] = dict(sorted(

